[bug] Build error on Windows due to libsamplerate-sys and CMAKE version

**describe the bug**
When trying to build from main, the `cargo build --release` command fails with the following message:

```bash
Compiling http-cache-reqwest v0.15.1
   Compiling sqlx-macros v0.7.2
   Compiling sentry v0.36.0
error: failed to run custom build command for `libsamplerate-sys v0.1.12`

Caused by:
  process didn't exit successfully: `C:\dev\screenpipe\target\release\build\libsamplerate-sys-12a5a6229b00932e\build-script-build` (exit code: 101)
  --- stdout
  CMAKE_TOOLCHAIN_FILE_x86_64-pc-windows-msvc = None
  CMAKE_TOOLCHAIN_FILE_x86_64_pc_windows_msvc = None
  HOST_CMAKE_TOOLCHAIN_FILE = None
  CMAKE_TOOLCHAIN_FILE = None
  CMAKE_GENERATOR_x86_64-pc-windows-msvc = None
  CMAKE_GENERATOR_x86_64_pc_windows_msvc = None
  HOST_CMAKE_GENERATOR = None
  CMAKE_GENERATOR = None
  CMAKE_PREFIX_PATH_x86_64-pc-windows-msvc = None
  CMAKE_PREFIX_PATH_x86_64_pc_windows_msvc = None
  HOST_CMAKE_PREFIX_PATH = None
  CMAKE_PREFIX_PATH = None
  CMAKE_x86_64-pc-windows-msvc = None
  CMAKE_x86_64_pc_windows_msvc = None
  HOST_CMAKE = None
  CMAKE = None
  running: "cmake" "C:\\Users\\vitor\\.cargo\\registry\\src\\index.crates.io-1949cf8c6b5b557f\\libsamplerate-sys-0.1.12\\libsamplerate" "-G" "Visual Studio 17 2022" "-Thost=x64" "-Ax64" "-DLIBSAMPLERATE_TESTS=OFF" "-DLIBSAMPLERATE_EXAMPLES=OFF" "-DLIBSAMPLERATE_INSTALL=OFF" "-DCMAKE_INSTALL_PREFIX=C:\\dev\\screenpipe\\target\\release\\build\\libsamplerate-sys-671474b33f3b1a26\\out" "-DCMAKE_C_FLAGS= -nologo -MT -Brepro" "-DCMAKE_C_FLAGS_MINSIZEREL= -nologo -MT -Brepro" "-DCMAKE_CXX_FLAGS= -nologo -MT -Brepro" "-DCMAKE_CXX_FLAGS_MINSIZEREL= -nologo -MT -Brepro" "-DCMAKE_ASM_FLAGS= -nologo -MT -Brepro" "-DCMAKE_ASM_FLAGS_MINSIZEREL= -nologo -MT -Brepro" "-DCMAKE_BUILD_TYPE=MinSizeRel"
  -- Configuring incomplete, errors occurred!

  --- stderr
  CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
    Compatibility with CMake < 3.5 has been removed from CMake.

    Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
    to tell CMake that the project requires at least <min> but has been updated
    to work with policies introduced by <max> or earlier.

    Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.

  thread 'main' panicked at C:\Users\vitor\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\cmake-0.1.54\src\lib.rs:1119:5:

  command did not execute successfully, got: exit code: 1

  build script failed, must exit now
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
```

I saw that there was a bug opened for macos, but his is regarding Windows 11 now.

**system info**
- os: Windows 11
- screenpipe version: v0.2.74

**additional context**

```
(base) PS C:\dev\screenpipe> cargo build --release
warning: patch for `hf-hub` uses the features mechanism. default-features and features will not take effect because the patch dependency does not support this mechanism
    Updating git repository `https://github.com/divanshu-go/libsamplerate-sys.git`
    Updating git submodule `https://github.com/leetal/ios-cmake`
    Updating git submodule `https://github.com/erikd/libsamplerate`
    Updating git repository `https://github.com/divanshu-go/rust-samplerate.git`
     Locking 2 packages to latest compatible versions
      Adding libsamplerate-sys v0.1.12 (https://github.com/divanshu-go/libsamplerate-sys.git?rev=6545fdd165673fce30d4ed5e82b845adc7b2faac#6545fdd1)
      Adding samplerate v0.2.4 (https://github.com/divanshu-go/rust-samplerate.git?rev=bf8415d5ac33940bb5a52bd23b24ca9a56f96792#bf8415d5)
   Compiling proc-macro2 v1.0.94
   Compiling unicode-ident v1.0.18
   Compiling serde v1.0.219
   Compiling autocfg v1.4.0
   Compiling windows_x86_64_msvc v0.52.6
   Compiling cfg-if v1.0.0
   Compiling shlex v1.3.0
   Compiling jobserver v0.1.32
   Compiling once_cell v1.21.1
   Compiling version_check v0.9.5
   Compiling libm v0.2.11
   Compiling log v0.4.26
   Compiling memchr v2.7.4
   Compiling itoa v1.0.15
   Compiling smallvec v1.14.0
   Compiling zerocopy v0.8.24
   Compiling pin-project-lite v0.2.16
   Compiling getrandom v0.3.2
   Compiling cc v1.2.17
   Compiling typenum v1.18.0
   Compiling serde_json v1.0.140
   Compiling parking_lot_core v0.9.10
   Compiling thiserror v1.0.69
   Compiling generic-array v0.14.7
   Compiling crossbeam-utils v0.8.21
   Compiling windows-targets v0.52.6
   Compiling num-traits v0.2.19
   Compiling lock_api v0.4.12
   Compiling slab v0.4.9
   Compiling scopeguard v1.2.0
   Compiling getrandom v0.2.15
   Compiling futures-core v0.3.31
   Compiling pkg-config v0.3.32
   Compiling stable_deref_trait v1.2.0
   Compiling windows-sys v0.52.0
   Compiling windows-sys v0.59.0
   Compiling tracing-core v0.1.33
   Compiling quote v1.0.40
   Compiling ryu v1.0.20
   Compiling futures-sink v0.3.31
   Compiling syn v2.0.100
   Compiling parking_lot v0.12.3
   Compiling allocator-api2 v0.2.21
   Compiling bitflags v1.3.2
   Compiling fnv v1.0.7
   Compiling paste v1.0.15
   Compiling crossbeam-epoch v0.9.18
   Compiling foldhash v0.1.5
   Compiling equivalent v1.0.2
   Compiling futures-channel v0.3.31
   Compiling hashbrown v0.15.2
   Compiling pin-utils v0.1.0
   Compiling crossbeam-deque v0.8.6
   Compiling futures-io v0.3.31
   Compiling num-integer v0.1.46
   Compiling rayon-core v1.12.1
   Compiling futures-task v0.3.31
   Compiling rand_core v0.9.3
   Compiling litemap v0.7.5
   Compiling writeable v0.5.5
   Compiling httparse v1.10.1
   Compiling ppv-lite86 v0.2.21
   Compiling percent-encoding v2.3.1
   Compiling subtle v2.6.1
   Compiling icu_locid_transform_data v1.5.0
   Compiling icu_properties_data v1.5.0
   Compiling form_urlencoded v1.2.1
   Compiling libc v0.2.164
   Compiling rand_chacha v0.9.0
   Compiling byteorder v1.5.0
   Compiling ring v0.17.14
   Compiling write16 v1.0.0
   Compiling utf8_iter v1.0.4
   Compiling utf16_iter v1.0.5
   Compiling icu_normalizer_data v1.5.0
   Compiling native-tls v0.2.14
   Compiling rand v0.9.0
   Compiling cpufeatures v0.2.17
   Compiling crypto-common v0.1.6
   Compiling block-buffer v0.10.4
   Compiling rand_core v0.6.4
   Compiling lazy_static v1.5.0
   Compiling arrayvec v0.7.6
   Compiling digest v0.10.7
   Compiling bytes v1.10.1
   Compiling anyhow v1.0.97
   Compiling windows-link v0.1.1
   Compiling rand_distr v0.5.1
   Compiling tower-service v0.3.3
   Compiling reborrow v0.5.5
   Compiling hex v0.4.3
   Compiling winapi v0.3.9
   Compiling rand_chacha v0.3.1
   Compiling try-lock v0.2.5
   Compiling httpdate v1.0.3
   Compiling rand v0.8.5
   Compiling want v0.3.1
   Compiling crc32fast v1.4.2
   Compiling encoding_rs v0.8.35
   Compiling seq-macro v0.3.6
   Compiling minimal-lexical v0.2.1
   Compiling simd-adler32 v0.3.7
   Compiling aho-corasick v1.1.3
   Compiling rustfft v6.2.0
   Compiling nom v7.1.3
   Compiling strength_reduce v0.2.4
   Compiling regex-syntax v0.8.5
   Compiling synstructure v0.13.1
   Compiling powerfmt v0.2.0
   Compiling deranged v0.4.0
   Compiling mio v1.0.3
   Compiling socket2 v0.5.8
   Compiling transpose v0.2.3
   Compiling primal-check v0.3.4
   Compiling time-core v0.1.4
   Compiling atomic-waker v1.1.2
   Compiling num-conv v0.1.0
   Compiling sha1 v0.10.6
   Compiling cmake v0.1.54
   Compiling time v0.3.40
   Compiling regex-automata v0.4.9
   Compiling tokio v1.42.0
   Compiling ahash v0.8.11
   Compiling bitflags v2.9.0
   Compiling adler2 v2.0.0
   Compiling untrusted v0.9.0
   Compiling miniz_oxide v0.8.5
   Compiling schannel v0.1.27
   Compiling serde_derive v1.0.219
   Compiling zerofrom-derive v0.1.6
   Compiling yoke-derive v0.7.5
   Compiling displaydoc v0.2.5
   Compiling bytemuck_derive v1.9.2
   Compiling zerovec-derive v0.10.3
   Compiling icu_provider_macros v1.5.0
   Compiling tracing-attributes v0.1.28
   Compiling thiserror-impl v1.0.69
   Compiling tokio-macros v2.4.0
   Compiling futures-macro v0.3.31
   Compiling num-bigint v0.4.6
   Compiling unicase v2.8.1
   Compiling zerofrom v0.1.6
   Compiling yoke v0.7.5
   Compiling bytemuck v1.22.0
   Compiling pulp v0.21.4
   Compiling futures-util v0.3.31
   Compiling zerovec v0.10.4
   Compiling regex v1.11.1
   Compiling num-complex v0.4.6
   Compiling half v2.5.0 (https://github.com/starkat99/half-rs.git?tag=v2.5.0#989d2700)
   Compiling tracing v0.1.41
   Compiling glob v0.3.2
   Compiling num-rational v0.4.2
   Compiling rustls v0.23.25
   Compiling tinystr v0.7.6
   Compiling icu_collections v1.5.0
   Compiling base64 v0.22.1
   Compiling mime v0.3.17
   Compiling icu_locid v1.5.0
   Compiling clang-sys v1.8.1
   Compiling flate2 v1.1.0
   Compiling http v1.3.1
   Compiling sha2 v0.10.8
   Compiling icu_provider v1.5.0
   Compiling icu_locid_transform v1.5.0
   Compiling tower-layer v0.3.3
   Compiling futures-executor v0.3.31
   Compiling prettyplease v0.2.31
   Compiling dyn-stack v0.13.0
   Compiling raw-cpuid v11.5.0
   Compiling zstd-sys v2.0.15+zstd.1.5.7
   Compiling sync_wrapper v1.0.2
   Compiling rustls-pki-types v1.11.0
   Compiling icu_properties v1.5.1
   Compiling windows_x86_64_msvc v0.48.5
   Compiling windows_x86_64_msvc v0.53.0
   Compiling pulp v0.18.22
   Compiling dyn-stack v0.10.0
   Compiling async-trait v0.1.88
   Compiling either v1.15.0
   Compiling indexmap v2.8.0
   Compiling rayon v1.10.0
   Compiling serde_urlencoded v0.7.1
   Compiling chrono v0.4.40
   Compiling icu_normalizer v1.5.0
   Compiling symphonia-core v0.5.4
   Compiling uuid v1.16.0
   Compiling raw-cpuid v10.7.0
   Compiling idna_adapter v1.2.0
   Compiling idna v1.0.3
   Compiling http-body v1.0.1
   Compiling windows-result v0.2.0
   Compiling http-body-util v0.1.3
   Compiling url v2.5.4
   Compiling vcpkg v0.2.15
   Compiling utf8parse v0.2.2
   Compiling gemm-common v0.18.2
   Compiling anstyle v1.0.10
   Compiling anstyle-parse v0.2.6
   Compiling gemm-common v0.17.1
   Compiling anstyle-wincon v3.0.7
   Compiling windows-strings v0.1.0
   Compiling futures v0.3.31
   Compiling anstyle-query v1.1.2
   Compiling winapi-util v0.1.9
   Compiling libloading v0.8.6
   Compiling is_terminal_polyfill v1.70.1
   Compiling colorchoice v1.0.3
   Compiling windows-targets v0.48.5
   Compiling windows-targets v0.53.0
   Compiling symphonia-metadata v0.5.4
   Compiling debugid v0.8.0
   Compiling anstream v0.6.18
   Compiling windows-implement v0.59.0
   Compiling windows-interface v0.59.1
   Compiling cexpr v0.6.0
   Compiling windows-result v0.3.2
   Compiling windows-strings v0.3.1
   Compiling toml_datetime v0.6.8
   Compiling winnow v0.7.4
   Compiling built v0.7.7
   Compiling aligned-vec v0.5.0
   Compiling crc-catalog v2.4.0
   Compiling v_frame v0.3.8
   Compiling tokio-util v0.7.14
   Compiling tower v0.5.2
   Compiling windows-core v0.59.0
   Compiling crc v3.2.1
   Compiling sentry-types v0.36.0
   Compiling rav1e v0.7.1
   Compiling windows-sys v0.48.0
   Compiling mime_guess v2.0.5
   Compiling profiling-procmacros v1.0.16
   Compiling h2 v0.4.8
   Compiling tokio-stream v0.1.17
   Compiling bzip2-sys v0.1.13+1.0.8
   Compiling data-encoding v2.8.0
   Compiling ident_case v1.0.1
   Compiling heck v0.5.0
   Compiling toml_edit v0.22.24
   Compiling regex-syntax v0.6.29
   Compiling clap_lex v0.7.4
   Compiling zerocopy v0.7.35
   Compiling ipnet v2.11.0
   Compiling strsim v0.11.1
   Compiling utf-8 v0.7.6
   Compiling base64 v0.21.7
   Compiling semver v1.0.26
   Compiling darling_core v0.20.10
   Compiling clap_builder v4.5.32
   Compiling sentry-core v0.36.0
   Compiling regex-automata v0.1.10
   Compiling proc-macro-crate v3.3.0
   Compiling clap_derive v4.5.32
   Compiling profiling v1.0.16
   Compiling hyper v1.6.0
   Compiling windows v0.59.0
   Compiling av1-grain v0.2.3
   Compiling tokio-native-tls v0.3.1
   Compiling same-file v1.0.6
   Compiling gemm-f32 v0.18.2
   Compiling maybe-rayon v0.1.1
   Compiling itertools v0.12.1
   Compiling arg_enum_proc_macro v0.3.4
   Compiling zeroize_derive v1.4.2
   Compiling num-derive v0.4.2
   Compiling hyper-util v0.1.10
   Compiling simd_helpers v0.1.0
   Compiling overload v0.1.1
   Compiling unicode_categories v0.1.1
   Compiling bitstream-io v2.6.0
   Compiling imgref v1.11.0
   Compiling noop_proc_macro v0.3.0
   Compiling thiserror v2.0.12
   Compiling weezl v0.1.8
   Compiling new_debug_unreachable v1.0.6
   Compiling quick-error v2.0.1
   Compiling hashbrown v0.14.5
   Compiling loop9 v0.1.5
   Compiling hyper-tls v0.6.0
   Compiling zeroize v1.8.1
   Compiling nu-ansi-term v0.46.0
   Compiling darling_macro v0.20.10
   Compiling gemm-f16 v0.18.2
   Compiling clap v4.5.32
   Compiling walkdir v2.5.0
   Compiling num_enum_derive v0.7.3
   Compiling matchers v0.1.0
   Compiling symphonia-utils-xiph v0.5.4
   Compiling libsqlite3-sys v0.26.0
   Compiling windows-registry v0.2.0
   Compiling gemm-f32 v0.17.1
   Compiling gemm-f64 v0.18.2
   Compiling gemm-c32 v0.18.2
   Compiling gemm-c64 v0.18.2
   Compiling http v0.2.12
   Compiling rustls-pemfile v2.2.0
   Compiling thiserror-impl v2.0.12
   Compiling structmeta-derive v0.3.0
   Compiling miette-derive v5.10.0
   Compiling fdeflate v0.3.7
   Compiling zune-inflate v0.2.54
   Compiling hmac v0.12.1
   Compiling avif-serialize v0.8.3
   Compiling sharded-slab v0.1.7
   Compiling inout v0.1.4
   Compiling lzma-sys v0.1.20
   Compiling num-iter v0.1.45
   Compiling tracing-log v0.2.0
   Compiling thread_local v1.1.8
   Compiling rustix v0.38.44
   Compiling lebe v0.5.2
   Compiling zip v1.1.4
   Compiling byteorder-lite v0.1.0
   Compiling rustversion v1.0.20
   Compiling rustc-demangle v0.1.24
   Compiling jpeg-decoder v0.3.1
   Compiling rustls v0.21.12
   Compiling fastrand v2.3.0
   Compiling unicode-width v0.1.14
   Compiling num_cpus v1.16.0
   Compiling zstd-safe v7.2.4
   Compiling zune-core v0.4.12
   Compiling bit_field v0.10.2
   Compiling rgb v0.8.50
   Compiling color_quant v1.1.0
   Compiling portable-atomic v1.11.0
   Compiling exr v1.73.0
   Compiling tiff v0.9.1
   Compiling ravif v0.11.11
   Compiling zune-jpeg v0.4.14
   Compiling gif v0.13.1
   Compiling miette v5.10.0
   Compiling sqlformat v0.2.6
   Compiling tempfile v3.17.1
   Compiling backtrace v0.3.74
   Compiling image-webp v0.2.1
   Compiling tracing-subscriber v0.3.19
   Compiling num v0.4.3
   Compiling cipher v0.4.4
   Compiling gemm v0.18.2
   Compiling png v0.17.16
   Compiling num_enum v0.7.3
   Compiling gemm-f16 v0.17.1
   Compiling reqwest v0.12.12
   Compiling crossbeam-queue v0.3.12
   Compiling rustc_version v0.4.1
   Compiling rustls-webpki v0.103.0
   Compiling futures-intrusive v0.5.0
   Compiling darling v0.20.10
   Compiling hashlink v0.8.4
   Compiling atoi v2.0.0
   Compiling spin v0.9.8
   Compiling gemm-c32 v0.17.1
   Compiling gemm-c64 v0.17.1
   Compiling gemm-f64 v0.17.1
   Compiling safetensors v0.4.5
   Compiling sct v0.7.1
   Compiling rustls-webpki v0.101.7
   Compiling qoi v0.4.1
   Compiling errno v0.3.10
   Compiling sha-1 v0.10.1
   Compiling onig_sys v69.8.1
   Compiling findshlibs v0.10.2
   Compiling memmap2 v0.9.5
   Compiling windows-core v0.52.0
   Compiling matrixmultiply v0.3.9
   Compiling bindgen v0.71.1
   Compiling event-listener v2.5.3
   Compiling syn v1.0.109
   Compiling dasp_sample v0.11.0
   Compiling xxhash-rust v0.8.15
   Compiling zstd-safe v5.0.2+zstd.1.5.2
   Compiling unicode-width v0.2.0
   Compiling dotenvy v0.15.7
   Compiling option-ext v0.2.0
   Compiling encode_unicode v1.0.0
   Compiling dirs-sys v0.4.1
   Compiling ssri v9.2.0
   Compiling sqlx-core v0.7.2
   Compiling console v0.15.11
   Compiling windows v0.52.0
   Compiling ug v0.1.0
   Compiling image v0.25.5
   Compiling socks v0.3.4
   Compiling gemm v0.17.1
   Compiling flume v0.11.1
   Compiling derive_builder_core v0.20.2
   Compiling sentry-contexts v0.36.0
   Compiling aes v0.8.4
   Compiling sentry-backtrace v0.36.0
   Compiling tungstenite v0.24.0
   Compiling webpki-roots v0.26.8
   Compiling http-serde v2.1.1
   Compiling itertools v0.13.0
   Compiling filetime v0.2.25
   Compiling home v0.5.11
   Compiling esaxx-rs v0.1.10
   Compiling windows-result v0.1.2
   Compiling number_prefix v0.4.0
   Compiling bit-vec v0.6.3
   Compiling rustc-hash v2.1.1
   Compiling base64ct v1.7.3
   Compiling bindgen v0.69.5
   Compiling rawpointer v0.2.1
   Compiling memmap2 v0.5.10
   Compiling zip v2.4.2
   Compiling lockfree-object-pool v0.1.6
   Compiling unicode-segmentation v1.12.0
   Compiling bumpalo v3.17.0
   Compiling password-hash v0.4.2
   Compiling heck v0.4.1
   Compiling zopfli v0.8.1
   Compiling bit-set v0.5.3
   Compiling indicatif v0.17.11
   Compiling windows-core v0.54.0
   Compiling ureq v2.12.1
   Compiling which v4.4.2
   Compiling tar v0.4.44
   Compiling sqlx-sqlite v0.7.2
   Compiling http-cache-semantics v2.1.0
   Compiling tokio-tungstenite v0.24.0
   Compiling axum-core v0.4.5
   Compiling derive_builder_macro v0.20.2
   Compiling candle-core v0.8.3 (https://github.com/huggingface/candle.git?rev=c930ab7e1a234f02a0f49350bf38f03f45e53757#c930ab7e)
   Compiling zstd v0.13.3
   Compiling hostname v0.4.0
   Compiling openapiv3-extended v6.0.0
   Compiling dirs v5.0.1
   Compiling structmeta v0.3.0
   Compiling pbkdf2 v0.12.2
   Compiling bzip2 v0.5.2
   Compiling lzma-rs v0.3.0
   Compiling itertools v0.11.0
   Compiling bincode v1.3.3
   Compiling os_info v3.10.0
   Compiling serde_path_to_error v0.1.17
   Compiling monostate-impl v0.1.14
   Compiling fs_extra v1.3.0
   Compiling reflink-copy v0.1.22
   Compiling base64 v0.13.1
   Compiling macro_rules_attribute-proc_macro v0.2.0
   Compiling constant_time_eq v0.3.1
   Compiling cacache v13.1.0
   Compiling plotters-backend v0.3.7
   Compiling inventory v0.3.20
   Compiling deflate64 v0.1.9
   Compiling matchit v0.7.3
   Compiling rustc-hash v1.1.0
   Compiling lazycell v1.3.0
   Compiling ciborium-io v0.2.2
   Compiling winsafe v0.0.19
   Compiling ciborium-ll v0.2.2
   Compiling axum v0.7.9
   Compiling candle-nn v0.8.3 (https://github.com/huggingface/candle.git?rev=c930ab7e1a234f02a0f49350bf38f03f45e53757#c930ab7e)
   Compiling macro_rules_attribute v0.2.0
   Compiling http-cache v0.20.1
   Compiling oasgen-core v0.22.0
   Compiling plotters-svg v0.3.7
   Compiling spm_precompiled v0.1.4
   Compiling whisper-rs-sys v0.12.1 (https://github.com/tazz4843/whisper-rs.git?rev=e0597486400ec436669e6ee3d8cc94b3859355f5#e0597486)
   Compiling onig v6.4.0
   Compiling monostate v0.1.14
   Compiling which v6.0.3
   Compiling sqlx-macros-core v0.7.2
   Compiling rayon-cond v0.3.0
   Compiling zstd v0.11.2+zstd.1.5.2
   Compiling sentry-debug-images v0.36.0
   Compiling ort-sys v2.0.0-rc.8
   Compiling derive_builder v0.20.2
   Compiling windows v0.54.0
   Compiling fancy-regex v0.13.0
   Compiling pbkdf2 v0.11.0
   Compiling sentry-panic v0.36.0
   Compiling sentry-tracing v0.36.0
   Compiling tokio-rustls v0.24.1
   Compiling reqwest-middleware v0.4.1
   Compiling h2 v0.3.26
   Compiling http-body v0.4.6
   Compiling bzip2 v0.4.4
   Compiling itertools v0.10.5
   Compiling serde_plain v1.0.2
   Compiling uiautomation_derive v0.4.1
   Compiling libsamplerate-sys v0.1.12
   Compiling is-terminal v0.4.16
   Compiling serde_derive_internals v0.29.1
   Compiling sqlite-vec v0.1.6
   Compiling unicode-normalization-alignments v0.1.12
   Compiling cpal v0.15.3
   Compiling eyre v0.6.12
   Compiling unsafe-libyaml v0.2.11
   Compiling cast v0.3.0
   Compiling extended v0.1.0
   Compiling tinyvec_macros v0.1.1
   Compiling constant_time_eq v0.1.5
   Compiling tinyvec v1.9.0
   Compiling zip v0.6.6
   Compiling knf-rs-sys v0.2.4 (https://github.com/Neptune650/knf-rs.git#b782e6bb)
   Compiling symphonia-format-riff v0.5.4
   Compiling criterion-plot v0.5.0
   Compiling oasgen-macro v0.22.0
   Compiling serde_yaml v0.9.34+deprecated
   Compiling hyper v0.14.32
   Compiling uiautomation v0.16.1
   Compiling tokenizers v0.21.1
   Compiling candle-transformers v0.8.3 (https://github.com/huggingface/candle.git?rev=c930ab7e1a234f02a0f49350bf38f03f45e53757#c930ab7e)
   Compiling http-cache-reqwest v0.15.1
   Compiling sqlx-macros v0.7.2
   Compiling sentry v0.36.0
error: failed to run custom build command for `libsamplerate-sys v0.1.12`

Caused by:
  process didn't exit successfully: `C:\dev\screenpipe\target\release\build\libsamplerate-sys-12a5a6229b00932e\build-script-build` (exit code: 101)
  --- stdout
  CMAKE_TOOLCHAIN_FILE_x86_64-pc-windows-msvc = None
  CMAKE_TOOLCHAIN_FILE_x86_64_pc_windows_msvc = None
  HOST_CMAKE_TOOLCHAIN_FILE = None
  CMAKE_TOOLCHAIN_FILE = None
  CMAKE_GENERATOR_x86_64-pc-windows-msvc = None
  CMAKE_GENERATOR_x86_64_pc_windows_msvc = None
  HOST_CMAKE_GENERATOR = None
  CMAKE_GENERATOR = None
  CMAKE_PREFIX_PATH_x86_64-pc-windows-msvc = None
  CMAKE_PREFIX_PATH_x86_64_pc_windows_msvc = None
  HOST_CMAKE_PREFIX_PATH = None
  CMAKE_PREFIX_PATH = None
  CMAKE_x86_64-pc-windows-msvc = None
  CMAKE_x86_64_pc_windows_msvc = None
  HOST_CMAKE = None
  CMAKE = None
  running: "cmake" "C:\\Users\\vitor\\.cargo\\registry\\src\\index.crates.io-1949cf8c6b5b557f\\libsamplerate-sys-0.1.12\\libsamplerate" "-G" "Visual Studio 17 2022" "-Thost=x64" "-Ax64" "-DLIBSAMPLERATE_TESTS=OFF" "-DLIBSAMPLERATE_EXAMPLES=OFF" "-DLIBSAMPLERATE_INSTALL=OFF" "-DCMAKE_INSTALL_PREFIX=C:\\dev\\screenpipe\\target\\release\\build\\libsamplerate-sys-671474b33f3b1a26\\out" "-DCMAKE_C_FLAGS= -nologo -MT -Brepro" "-DCMAKE_C_FLAGS_MINSIZEREL= -nologo -MT -Brepro" "-DCMAKE_CXX_FLAGS= -nologo -MT -Brepro" "-DCMAKE_CXX_FLAGS_MINSIZEREL= -nologo -MT -Brepro" "-DCMAKE_ASM_FLAGS= -nologo -MT -Brepro" "-DCMAKE_ASM_FLAGS_MINSIZEREL= -nologo -MT -Brepro" "-DCMAKE_BUILD_TYPE=MinSizeRel"
  -- Configuring incomplete, errors occurred!

  --- stderr
  CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
    Compatibility with CMake < 3.5 has been removed from CMake.

    Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
    to tell CMake that the project requires at least <min> but has been updated
    to work with policies introduced by <max> or earlier.

    Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.



  thread 'main' panicked at C:\Users\vitor\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\cmake-0.1.54\src\lib.rs:1119:5:

  command did not execute successfully, got: exit code: 1

  build script failed, must exit now
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
```

--- a/.cargo/config.toml
+++ b/.cargo/config.toml
@@ -18,3 +18,6 @@ rustflags = ["-C", "link-arg=-Wl,--allow-multiple-definition"]
 
 [target.aarch64-apple-darwin]
 rustflags = "-lc++ -l framework=Accelerate"
+
+[env]
+CMAKE_POLICY_VERSION_MINIMUM = "3.5"

